### DISCUSSION TITLE
Coupe d'Allemagne non trouvée sur Netbet

### DISCUSSION THREAD

**Discussion by AoooR42** [pretrehr/Sports-betting#9] [General]
Bonjour, la Coupe d'Allemagne n'est pas trouvée sur netbet, alors qu'elle existe sous le nom "dfb-pokal" (ici : https://www.netbet.fr/football/allemagne/dfb-pokal)
![image](https://user-images.githubusercontent.com/77069582/104312816-6c809500-54d7-11eb-9e0b-030f4a9e2df6.png)

**Comment by pretrehr** [comment]
Bonjour,
Merci pour votre message, le problème devrait être résolu maintenant.
